.circleci/config.yml

```diff
@@ -76,6 +77,7 @@ cache_2: &cache_2
     - packages/opentelemetry-exporter-stackdriver-trace/node_modules
 
 node_unit_tests: &node_unit_tests
+  resource_class: large
```

Comment:
I think the default is medium 2 vCPUs and 4GB RAM, are you saying we need more resources to run the tests?

Reply:
Yes, the tests failed using npm on the default resource class.

Reply:
Ok. We are sharing CircleCI resources with other OpenTelemetry languages, I think we should be mindful about that.